### PR TITLE
feat(TabNews): add presence

### PR DESCRIPTION
**Pull request by Kuriel23**
## Description 
Presence for TabNews, a forum for programming and technology

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image

</details>

### CODE REVIEW

File: websites/T/TabNews/metadata.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "354233941550694400",
+		"name": "Kuriel#1411"
```
Comment (theusaf) [inline websites/T/TabNews/metadata.json:5]:
```suggestion
		"name": "Kuriel"
```

File: websites/T/TabNews/presence.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	const presenceData: PresenceData = {
+			details: "Outros",
+			largeImageKey: "https://i.imgur.com/pQeO9sn.png",
+			startTimestamp: browsingTimestamp,
+			buttons: [
+				{
+					label: "Ver conteúdo",
+					url: document.location.href,
+				},
+			],
```
Comment (theusaf) [inline websites/T/TabNews/presence.ts:16]: Buttons that can lead to home pages are [not allowed](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets). Generally, buttons should lead to more specific content.
```suggestion
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by Kuriel23** [comment]
Done!

File: websites/T/TabNews/metadata.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	},
+	"service": "TabNews",
+	"description": {
+		"en": "TabNews is a website with content for those who work with Programming and Technology.",
+		"pt_BR": "TabNews é um site com conteúdo para quem trabalha com Programação e Tecnologia."
+	},
+	"url": "www.tabnews.com.br",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/YlPvIIb.png",
+	"thumbnail": "https://i.imgur.com/vUlsC33.png",
```
Comment (plskz) [inline websites/T/TabNews/metadata.json:15]: use this

```suggestion
	"thumbnail": "https://i.imgur.com/hbQ4bpd.png",
```

**Review by plskz — CHANGES_REQUESTED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by Timeraa — APPROVED** [review]

**Pull request merged**